public/assets/sass/style.scss

```diff
@@ -72,6 +72,39 @@ body {
   background: $color1;
 }
 
+kb-code-editor {
```

Comment:
I moved all of this to the `_codeEditor.scss` new file.

so fix this there please and remove it entirely from here (or do a merge if you haven't done so yet)